CHANGES/111.feature.rst

```diff
@@ -0,0 +1 @@
+1421.feature.rst
```

Comment:
Sorry, I've missed. How is this file rendered finally? What is desired CHANGELOG record?

Reply:
<img width="702" alt="Screenshot 2024-11-18 at 9 28 03 PM" src="https://github.com/user-attachments/assets/d321cb81-decc-46f0-9607-bce88ea9fa56">

https://yarl--1421.org.readthedocs.build/en/1421/changes/#features

Reply:
Usually I use the PR# for the changelog messages. If the issue has a lot in it, symlink to the PR as well so both show up

Reply:
Got you